salt/returners/smtp_return.py

```diff
@@ -226,14 +238,14 @@ def returner(ret):
                'Subject: {3}\r\n'
                '\r\n'
                '{4}').format(from_addr,
-                             to_addrs,
+                             ', '.join(to_addrs),
                              formatdate(localtime=True),
                              subject,
                              content)
 
     log.debug('smtp_return: Connecting to the server...')
     server = smtplib.SMTP(host, int(port))
-    server.set_debuglevel = 'debug'
+    server.set_debuglevel(1)  # enable logging SMTP session
```

Comment:
Does this expose username and passwords if SMTP authentication is enabled? I'm not sure I like this as a default.